package.json

```
@@ -77,13 +77,13 @@
     "authorization"
   ],
   "engines": {
-    "node": ">= 6.*"
+    "node": ">= 8.*"
```

Node 8 is already EOL. `ember-simple-auth@^3.0.0` also requires Node >= 10. Would recommend to drop support for node 8 as well to avoid another major release.

```suggestion
    "node": ">= 10.*"
```

---

I have reverted this to support node v6.